components/common/shoppingCart/ShoppingCartItemRow.vue

```diff
@@ -58,9 +64,15 @@ const avatar = ref<string>()
 const isHovered = useElementHover(hoverRef)
 const emit = defineEmits(['delete', 'click-item'])
 
-const props = defineProps<{
-  nft: ShoppingCartItem
-}>()
+const props = withDefaults(
+  defineProps<{
+    nft: ShoppingCartItem
+    round?: number
```

Comment:
I don't see there is anything from the parent component. Why do we need this prop?

Reply:
I left it open in case there was ever a need. I would make it a single number.